lib/spack/spack/test/cmd/install.py

```diff
@@ -741,6 +744,7 @@ def test_install_deps_then_package(tmpdir, mock_fetch, install_mockery):
     assert os.path.exists(root.prefix)
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Environments not supported on windows")
```

Comment:
Same for other lines like this... envs _are_ supported on Windows post Dan's PR landing, at least AFAIK, however views are NOT currently supported. If this is failing because of views, can we update the reason string, and if it's failing just on envs inherently, can you send me the error message?

Reply:
Good catch, most of them are failing rn because of the views. As far as I can tell, the symlink changes should make these pass. It's a bunch of the same error which I had resolved for the Environment tests PR.

Reply:
Ok great, lets maybe even call out that PR in the reason message? It'll be helpful to grep and re-enable things

Reply:
Makes sense. I added the PR No. in the message